frontend/src/components/static_pages/about.css

```diff
@@ -11,15 +11,30 @@
 .about-people-row {
   margin-bottom: 20px;
 }
-
 .images {
-  margin: 0;
-  margin-bottom: 40px;
-  margin-left: 120px;
+ margin: 0;
+ margin-bottom: 40px;
+ margin-left: 120px;
 }
 
 .images img {
-  width: 300px;
+ width: 330px;
```

Comment:
also a bit of an issue around the 1100px breakpoint 
![image](https://user-images.githubusercontent.com/1637481/68438797-ef4ae900-018a-11ea-866e-174d713523e2.png)